docs/admin/guides/build-image.md

```diff
@@ -7,23 +7,24 @@
 
 Users can add new images to a container repository by uploading a Containerfile. The syntax for
 Containerfile is the same as for a Dockerfile. The same REST API endpoint also accepts a JSON
-string that maps artifacts in Pulp to a filename. Any artifacts passed in are available inside the
+string that maps artifacts in Pulp to a filename. Any file passed in are available inside the
 build container at `/pulp_working_directory`.
```

Comment:
Is that still true? Didn't we say we remove the `artifacts` parameter and ask for a `build_context` instead?